library/src/main/java/br/com/goncalves/pugnotification/notification/Custom.java

```diff
@@ -91,7 +84,8 @@ public Custom textBackground(@ColorInt int color, boolean showGradient) {
 
         if (showGradient) {
             this.remoteViews.setViewVisibility(R.id.notification_img_bottom_gradient, View.VISIBLE);
-            this.remoteViews.setImageViewBitmap(R.id.notification_img_bottom_gradient, createGradientBitmap(color));
+            this.remoteViews.setImageViewResource(R.id.notification_img_bottom_gradient, R.drawable.img_notif_gradient);
+            this.remoteViews.setInt(R.id.notification_img_bottom_gradient, "setColorFilter", color);
```

Comment:
@kmalmur which `PorterDuff.Mode` is being chosen as a result of this?

Reply:
@blipinsk SRC_ATOP